CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Change Log
+* Modify connectionWidget to support authType for databases other than SQL Server
```

Comment:
@kburtram, I believe we generate the change log at release, we don't modify it as we go.

Reply:
I'll revert. Thanks.